src/docs/ganache/corda/transactions.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Ganache | Corda Transactions
+layout: docs.hbs
+---
+# Transactions
+
+<p class="alert alert-info">For Corda technical documentation not related to Ganache head over to [Corda's official documentation](https://docs.corda.net/docs/corda-os/4.4.html).</p>
+
+The Corda Transactions screen displays a list of all transactions, filterable by node/notary.
+
+![Corda Transactions](/img/docs/ganache/corda/transactions.png)
+
+Clicking on a transaction opens the Transaction Details screen. From this screen you can find which CorDapp flows were
```

Comment:
Just a suggestion:
Clicking on a transaction opens the Transaction Details screen. From this screen you can find which CorDapp flows, attachments, and input and output states were involved in the transaction.